pkg/cmd/operator/webhooks.go

```diff
@@ -175,12 +192,18 @@ func (o *WebhookOptions) Run(streams genericclioptions.IOStreams, commandName st
 	})
 	handler.Handle("/validate", admissionreview.NewHandler(validate))
 
+	if o.dynamicCertKeyPairContent != nil {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			o.dynamicCertKeyPairContent.Run(1, ctx.Done())
```

Comment:
It's unclear what `1` means, name it in const

Reply:
technically it's the name of the function argument that determines the meaning of that int, not the name of a variable you pass in so bare int has the same meaning, const is primarily used for something that needs to be set across the board or possibly changed but not to infer a "type"

Reply:
I cannot deduce what this 1 means without looking at the source code of third party library, nor from purpose of this library. It's clear for `WaitGroup.Add(1)`, but not for `DynamicCertKeyPairContent.Run(1, ...)`
Naming it here would make my (as a reviewer) life easier, because I would be able to read the code without going back and forth between IDE and browser/IDE tabs.

Reply:
how do you (as a reviewer) know that someone putting `const numberOfDays` into `DynamicCertKeyPairContent.Run(numberOfDays, ...)` is doing a correct job without looking at the function or knowing that conventionally it takes `numberOfWorkers`?

Reply:
When name is given i can at least judge if it make sense. `numberOfWorkers` in Run makes me think that there're workers - additional goroutines - and this stop channel probably stops them - and I would be right.  
Where `1` or `numberOfDays` doesn't make any sense.